components/post_view/post_attachment_opengraph/post_attachment_opengraph.tsx

```diff
@@ -41,230 +47,212 @@ export type Props = {
     isInPermalink?: boolean;
 };
 
-export default class PostAttachmentOpenGraph extends React.PureComponent<Props> {
-    getImageMetadata = (imageUrl?: string) => {
-        if (!imageUrl) {
-            return undefined;
-        }
-
-        const imagesMetadata = this.props.post.metadata.images;
-        if (!imagesMetadata) {
-            return undefined;
-        }
-
-        return imagesMetadata[imageUrl];
-    };
-
-    isLargeImage = (dimensions?: PostImage) => {
-        if (!dimensions) {
-            return false;
-        }
-
-        const {height, width} = dimensions;
-
-        const largeImageMinRatio = 16 / 9;
-        const largeImageMinWidth = 150;
+type ImageData = {
```

Comment:
What's this type for? Shouldn't this be either `OpenGraphMetadataImage` or `PostImage`, or were we mixing those up in this code before?

Reply:
they have been mixed apparently, but the code itself did provide several conditions to rule out the properties that were missing/extra, so it never resulted in bugs.